proactive-jupyter-kernel/kernel.py

```diff
@@ -886,21 +847,127 @@ def __create_task__(self, input_data):
+            if input_data['name'] in self.exported_vars:
+                del self.exported_vars[input_data['name']]
+
+            if 'language' in input_data:
+                if input_data['language'] in self.proactive_script_languages:
+                    self.__kernel_print_ok_message__('Changing script language to \'' + input_data['language']
+                                                     + '\' ...\n')
+                    proactive_task.setScriptLanguage(self.proactive_script_languages[input_data['language']])
+                else:
+                    raise ParameterError('Language \'' + input_data['language'] +
+                                         '\' not supported!\n Supported Languages:\n' + self.script_languages)
+            else:
+                self.__kernel_print_ok_message__('Changing script language to \'Python\' ...\n')
+                proactive_task.setScriptLanguage(self.proactive_script_languages['Python'])
+
+        else:
+            if 'language' in input_data:
+                if input_data['language'] in self.proactive_script_languages:
+                    self.__kernel_print_ok_message__('Creating a proactive ' + input_data['language'] + ' task ...\n')
+                    proactive_task = self.gateway.createTask(self.proactive_script_languages[input_data['language']])
+                elif input_data['language'] == 'Python':
+                    self.__kernel_print_ok_message__('Creating a proactive \'Python\' task ...\n')
+                    proactive_task = self.gateway.createPythonTask()
+                else:
+                    raise ParameterError('Language \'' + input_data['language'] +
+                                         '\' not supported!\n Supported Languages:\n' + self.script_languages)
+            else:
+                self.__kernel_print_ok_message__('Creating a proactive \'Python\' task ...\n')
+                proactive_task = self.gateway.createPythonTask()
+
+            if input_data['name'] == '':
+                name = self.__get_unique_task_name__()
+                self.__kernel_print_ok_message__('WARNING: Task \'' + input_data['name'] + '\' renamed to : '
+                                                 + name + '\n')
+                input_data['name'] = name
+
+            proactive_task.setTaskName(input_data['name'])
+
+            self.__kernel_print_ok_message__('Task \'' + input_data['name'] + '\' created.\n')
+
+            self.proactive_tasks.append(proactive_task)
+            self.tasks_names.append(input_data['name'])
+
+            self.tasks_count += 1
+
+        self.__set_default_selection_script__(proactive_task)
+        self.__set_default_fork_environment__(proactive_task)
+        self.__set_dependencies_from_input_data__(proactive_task, input_data)
+        self.__set_generic_information_from_input_data__(proactive_task, input_data)
+
+        # TODO: check how to import/export variables when a file path is provided
```

Comment:
avoid committing todos

Reply:
The TODOs that I commit are possible enhancements that will not be done now because not very important to the release. It is better to have them.